src/sagas/account.js

```diff
@@ -114,8 +129,17 @@ export function* loginFlow () {
   }
 }
 
+export function* userFlow () {
+  while(true) {
+    const action = yield take(PATCH_USER);
+
+    yield call(doPatchUser, action);
+  }
+}
+
 export default function* () {
   yield all([
     loginFlow(),
+    userFlow(),
```

Comment:
@chris-allen Good call. I adopted the pattern if putting these in a loop because I thought it would complete after it received the first `PATCH_USER` but I guess `takeEvery` keeps the saga alive listening for future actions?